theano/tensor/tests/test_basic.py

```diff
@@ -5174,7 +5169,7 @@ def test_flatten_outdim2():
     f = inplace_func([a], c)
     assert numpy.all(f(a_val) == a_val)
 
-    utt.verify_grad(Flatten(2), [a_val])
+    utt.verify_grad(flatten, [a_val])
```

Comment:
Oh, but ```Flatten(2)``` and ```flatten``` are not equivalent right? It will call ```flatten``` with default ```outdim=1``` while we wanted ```outdim=2```. You should use [functools.partial](https://docs.python.org/2/library/functools.html#functools.partial) to preset ```outdim=2```.